exonum-java-binding/core/rust/src/storage/proof_list_index.rs

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::{panic, ptr};
```

Comment:
Minor: we use "external, std, internal" imports order everywhere just for consistency, according to https://github.com/exonum/exonum/blob/master/CONTRIBUTING.md#conventions (though we're not following orders of `pub use` and `mod` from there, lol)